app/scripts/modules/core/src/managed/Environments.tsx

```diff
@@ -157,6 +159,13 @@ export function Environments({ app }: IEnvironmentsProps) {
             show && (
               <animated.div key={key} className={styles.environmentsPane} style={props}>
                 <ColumnHeader text="Environments" icon="environment" />
+                <EnvironmentsHeader
+                  app={app}
+                  clusterInfo={{
+                    managed: resources.filter(r => getKindName(r.kind) === 'cluster' && !r.isPaused).length,
+                    total: resources.filter(r => getKindName(r.kind) === 'cluster').length,
```

Comment:
as we move forward with our plans to evolve and add to the available kinds of resources, focusing only on cluster resources will become less and less relevant (and many apps may end up with _no_ cluster resources). Thoughts on shifting this to focus on resource in general of any kind vs. just clusters?

Reply:
Good call. I'm removing the filtering.